packages/frontend/Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY babel.config.js .
 COPY package.json .
 COPY packages/api/package.json packages/api/package.json
 COPY packages/frontend/package.json packages/frontend/package.json
+COPY packages/frontend-v2/package.json packages/frontend-v2/package.json
```

Comment:
we should probably figure out a way to just copy recursively all package.jsons listed in the workspace... 

[this blog post](https://www.huy.dev/yarn-v2-workspace-docker-vs-code-2020-03-23/) does this via [this weird command](https://github.com/huy-nguyen/yarn-v2-workspace-docker-vs-code/blob/6c427bf97a4a17ddc4bbb911b6c473c55d9667a3/Dockerfile#L18) – though feel free to just kick the can for later

Reply:
yea lmao, probably going to kick the can for later cause we're moving to the new frontend anyways... we can nail down the infra of our new stack once we're done(maybe not even use docker?!)